### PR TITLE
[Staging] Feedback 05

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

Address small changes before going live:

* [X] Add `number` to HURUmap stat type,
* [X] Introduce `RichTypography` component to the UI to handle automatic styling of return content from `RichTextField` using our `Theme`,
* [ ]  ~Make sure all/most of backend using `RichTextField` passes it through a `richtext` filter before returning to the front end. This will correctly translate images into proper `img` tags,~ This requires it's own PR since we're too close to launch to be making such a wholesale change.
* etc.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/takwimu_ui/src/components/Profile/index.js [hunk 1]
```diff
@@ -47,6 +47,12 @@ class Profile extends React.Component {
 
     const title = value === 0 ? 'Data by Topic' : tabs[value].name;
 
+    // Wagtail inserts div/p when RichTextField is empty
+    const hasDescription = () =>
+      description &&
+      description.length > 0 &&
+      description !== '<p></p>' &&
+      description !== '<div class="rich-text"></div>';
```
Comment (karimkawambwa) [inline takwimu/takwimu_ui/src/components/Profile/index.js]: why not just strip all html and then check length to be safe. `.replace(/<[^>]+>/g, '')` should work.

**Comment by phillipahereza** [comment]
@kilemensi update to HURUmap `0.6.2`

File: takwimu/takwimu_ui/src/components/Profile/index.js (hunk 1 above)
Comment (kilemensi) [reply takwimu/takwimu_ui/src/components/Profile/index.js]: [Regular Expresssion, Now You Have Two Problems](https://blog.codinghorror.com/regular-expressions-now-you-have-two-problems/) :sunglasses:

Comment (karimkawambwa) [reply takwimu/takwimu_ui/src/components/Profile/index.js]: I'll have to read this

**Pull request merged**